### PR TITLE
Add reference on DataFrames.jl, use print not println, capitalization

### PR DESCRIPTION
**Pull request by dmbates**
- Add reference and bib entry for Bouchet-Valat and Kaminski 2023, cited in Appendix A
- Use `print` instead of `println` in code blocks (where trailing newlines are not needed).
- Make capitalization in chapter titles uniform.  I chose treat the chapter title as a sentence.

### CODE REVIEW

File: intro.qmd [hunk 1]
```diff
@@ -320,11 +320,23 @@ By default, this compact display of mixed-effects model fits is used in [Jupyter
...
 ```
 
 The call to `fit` constructs a `LinearMixedModel` object, evaluates the [maximum likelihood](https://en.wikipedia.org/wiki/Maximum_likelihood_estimation) parameter estimates, assigns the results to the name `dsm01`, and displays a summary of the fitted model.
 
+:::{.callout-note collapse="true"}
+### The named argument, progress=false
+
+The optional, named argument, `progress=false`, suppresses the creation and display of a progress meter for the iterations.
+Such displays are useful in interactive sessions to monitor the progress of the iterations but not in a static display for a book.
+The comma used to separate argument values can be replaced by a semi-colon when it separates positional arguments, which must precede named arguments, from the first named argument.
```
Comment (palday) [inline intro.qmd:333]: I don't have a better phrasing suggestion at the moment, but I had to read this twice to understand it and I know the behavior in question.

File: intro.qmd [hunk 2]
```diff
@@ -851,29 +864,31 @@ When there are hundreds or thousands of random effects the `qqcaterpillar` form
...
 ```{julia}
 #| code-fold: false
 #| eval: false
 contrasts = Dict(:batch => Grouping())
 ```
 
+but current versions of the `MixedModels` package do this automatically.
+
+:::{.callout-note}
+### Replace "current versions" by a specific version number
```
Comment (palday) [inline intro.qmd:886]: based on NEWS, this was added in 4.21

File: longitudinal.qmd [hunk 3]
```diff
@@ -61,7 +61,8 @@ In this chapter we introduce graphical and statistical techniques for the analys
 
 ## The elstongrizzle data
 
-Data from a dental study measuring the lengths of the ramus bone (mm) in 20 boys at 8, 8.5, 9, and 9.5 years of age were reported in @elstongrizzle1962 and in @davis2002.
+Data from a dental study measuring the lengths of the [ramus of the mandible](https://en.wikipedia.org/wiki/Mandible#Ramus) (mm) in 20 boys at 8, 8.5, 9, and 9.5 years of age were reported in @elstongrizzle1962 and in @davis2002.
+(Following the description in both of these references we will refer to these as measurements of the "ramus bone", even though this is somewhat inaccurate.)
```
Comment (palday) [inline longitudinal.qmd:65]: :heart: 

much appreciated specification :)

**Review by palday — APPROVED** [review]

File: intro.qmd (hunk 1 above)
Comment (dmbates) [reply intro.qmd:333]: I modified the text to what I hope is a more comprehensible passage.

Comment (palday) [reply intro.qmd]: Yes, much better!

File: intro.qmd (hunk 2 above)
Comment (dmbates) [reply intro.qmd:887]: Thanks, now incorporated

**Pull request merged**